composer.json

```diff
@@ -5,7 +5,8 @@
     "license": "MIT",
     "require": {
         "php": "^8.2",
-        "phpstan/phpstan": "^1.10.50"
+        "phpstan/phpstan": "^1.10.50",
+        "nikic/php-parser": "^4"
```

Comment:
stick to php-parser v4 as otherwise v5 would be installed automatically and be in conflict with rector v1 and phpstan v1